spec/text-document-spec.coffee

```diff
@@ -2,28 +2,36 @@ Point = require "../src/point"
 TextDocument = require "../src/text-document"
 
 describe "TextDocument", ->
-  document = null
```

Comment:
In this spec we should actually not use the layer-based translation methods but instead test things in terms of the original `TextBuffer` API. If we have any tests in the original library for mapping row/column positions to character offsets we can copy them.